libs/atqr/domain/src/lib/domain-spec/stories/playerReceivesCondolences.disl

```diff
@@ -0,0 +1,13 @@
+read_also api.disl
+read_also enums.disl
+read_also ui.disl
+
+doman/NotificationService calls ChallengeFailed!
+    rules:
+        - JudgementStatus == Judged
+        - ChallengeStatus == Finished
+        - PlayerOutcome == Failed
+
+    when. 
+        Challenge.PlayerOutcome is Failed
```

Comment:
Challenge does not have this property (or at least shouldn't). This should com from `Enrollment`

Reply:
when. 
        Challenge.@players.PlayerOutcome is Failed
            ChallengeFailed!
            NotificationService calls ChallengeFailed! email
            
            like this ??

Reply:
can I say like this, since @players: Enrollment[] ? 

Challenge.@players.PlayerOutcome is Failed

Reply:
Almost, `Enrollment`'s property is `outcome`, not `PlayerOutcome` and you forgot the array index. The correct statement is `challenge.@players[0].outcome`. But you could have said the same in natural language: `challenge's  player outcome`

Reply:
ok ! thanks